tests/Dfc.CourseDirectory.WebV2.Tests/StubControllers/ProviderController.cs

```diff
@@ -6,8 +6,5 @@ public class ProviderController : Controller
     {
         [HttpGet("Provider/Dashboard")]
         public IActionResult Dashboard() => Ok();
-		
-        [HttpGet("provider/provider-type")]
-        public IActionResult AddOrEditProviderType([FromQuery] bool updateProviderType) => Ok();
```

Comment:
It's an admin-only action so I didn't feel that maintaining the existing URL was important, especially given that it's not (safely) bookmarkable because it has a dependency on a session value.